bazelbuild/rules_scala
# Add flag to disable writing volatile bits of statsfile

### Description
Writing the milliseconds bit of the statsfile can cause hash sum differences in the disk cache. Adding a flag to disable writing the timestamp in attempt in improving cache hit rate when distributed with remote caching. 

### Motivation
We are trying to improve the cache hit rate for the action graph. 

### Manual Test

Doing the following should yield zero differences between the different disk caches when `scala_toolchain` `enable_stats_file = False`

```
rm -rf $DC1 $DC2
DC1=/tmp/disk_cache_15
DC2=/tmp/disk_cache_16
TARGET=<sometarget>
mkdir -p $DC1 && mkdir -p $DC2 && ./bazel clean && ./bazel build --disk_cache=$DC1 $TARGET && ./bazel clean && ./bazel build --disk_cache=$DC2 $TARGET && diff -r $DC1 $DC2
```
